rs-client: Cleanup

Lucwousin opened the pull request.

ThatGamerBlue commented on runescape-client/src/main/java/Friend.java:
```diff
@@ -61,15 +67,14 @@ public int compareTo(Object var1) {
-	@Export("doesSpriteExist")
-	public static boolean doesSpriteExist(AbstractArchive var0, int var1, int var2) {
+	@Export("SpriteBuffer_loadSprite")
+	public static boolean SpriteBuffer_loadSprite(AbstractArchive var0, int var1, int var2) {
```
`SpriteBuffer_cacheSprite` maybe? Want to make it obvious this doesn't return a `Sprite` instance based off its name, as that's why it got broke in 180 -> 181

Lucwousin replied:
My vote is for bufferSprite. If I didn't have a problem with long method names I'd go for tryToLoadSpriteIntoBuffer but that's way too long imo. Maybe loadFile is even better actually, as it is loading pixels from a file

ThatGamerBlue replied:
+1 for `bufferSprite`

Lucwousin commented:
Missing exports don't matter. The only reason they go missing is cause updatemappings only adds export annotations on either the child/parent class methods, and the checkmappings test doesn't check for that. It makes things more consistent. And means you can be sure something is wrong if @ Export is missing.

I'll compare the bytecode of the original output of updatemappings, the current master version and this branch to see if things differ.

Lucwousin commented:
original bytecode: https://gist.github.com/Lucwousin/609230fadbf91bab30ef6032f67f605b
(has the opcodes replaced due to updatemappings doing that)
master bytecode: https://gist.github.com/Lucwousin/7c82dbf00b8546a585c7e5286abf49bb
modified bytecode: https://gist.github.com/Lucwousin/536b2abc00be4880b42f5ac40040271c

both master and modified have local var debug info, but that can be ignored.

My point is building rs-client already ends up with modified bytecode from the original. As long as the code functionally is the same (master and modified only really differ in block ordering) updatemappings can map them
